examples/jquery.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>

```

Comment:
I have dropped jQuery support in this repository. 
The main reason is that using jQuery doesn't contribute to reducing code or making the library easier to use.